assets/css/bpmn-js-bpmnlint.css

```
@@ -101,22 +112,32 @@
 
 .bjsl-issues .icon {
   display: flex;
+  width: var(--bjsl-icon-dimensions);
+  height: var(--bjsl-icon-dimensions);
 }
 
 .bjsl-issues li + li {
   margin-top: .5rem;
 }
 
-.bjsl-issues .error svg {
-  color: #cc3300;
+.bjsl-issues .icon {
+  color: var(--icon-color);
+}
+
+.bjsl-issues .error {
+  --icon-color: var(--bjsl-color-error);
 }
 
-.bjsl-issues .warning svg {
-  color: #f7c71a;
+.bjsl-issues .warning {
+  --icon-color: var(--bjsl-color-warning);

```

we are inconsistent in prefixing the css variables here. If we want to be safe, we should do 

```suggestion
  --bjsl-icon-color: var(--bjsl-color-warning);
```

as well, so any other `icon-color` variable does not influence the linting

---

Good catch. I did not want to prefix, because it is essentially a local variable:

```css
/* defines base */
.foo {
  --icon-color: green;
  color: var(--icon-color);
}

/* defines override */
.foo.special {
  --icon-color: yellow;
}
```

But the base variable is missing in this case. Will clean this up.